ledger/ledger-on-memory/BUILD.bazel

```diff
@@ -58,6 +58,7 @@ da_scala_test(
         "//ledger/participant-state",
         "//ledger/participant-state/kvutils",
         "//ledger/participant-state/kvutils:kvutils-tests-lib",
+        "//ledger/participant-state/kvutils/app",
```

Comment:
I'd appreciate it if we can move the test dependencies into `kvutils` rather than `kvutils/app`. If you need to keep the options reader in `app`, fair enough, but the configuration class can still be moved.

Reply:
Changed that -- now all the batching related config definition & factory classes are in `kvutils` and only the config parsing logic is in `kvutils/app`.